onnx/common/ir.h

```diff
@@ -866,6 +866,29 @@ friend struct Value;
 
   std::vector <OpSetID> opset_versions_;
 
+  bool isNameInAnyNode(const std::string& name) const {
```

Comment:
What happens when we have nested-scopes produced via control-flow operators? Don't you want to ensure uniqueness handles that too?